hardware/raspberrypi/revision-codes/README.md

```diff
@@ -86,34 +95,37 @@ uuuuuuuuFMMMCCCCPPPPTTTTTTTTRRRR
 |          |              | 11: 4B                     |
 | RRRR     | Revision     | 0, 1, 2, etc.              |
 
+<sup>1</sup> Information on programming the OTP bits can be found [here](../../industrial/README.md) and [here](../otpbits.md)
```

Comment:
Tiny nit: full stop missing from end of line

Reply:
Do footnotes need full stops? IMHO that's a question for the copy-editors.

Reply:
I have always assumed they do need a full stop, since they're a sentence (or fragment) in their own right. Our universities seem to think so as well - a quick google reveals  https://www.bristol.ac.uk/arts/exercises/referencing/page_08.htm,  https://www2.le.ac.uk/library/help/referencing/footnote and https://libguides.ials.sas.ac.uk/c.php?g=659237&p=4653506.

But this one is a bit nit-picky, even for me 😋